subhalo_properties.py

```diff
@@ -932,31 +934,34 @@ def veldisp_matrix_dm(self) -> unyt.unyt_array:
         )
 
     @lazy_property
-    def DM_Vmax(self) -> unyt.unyt_quantity:
+    def DM_Vmax_soft(self) -> unyt.unyt_quantity:
         """
         Maximum circular velocity of the dark matter particles in the subhalo.
+        Particles are set to have minimum radius equal to their softening length.
         """
         if self.Ndm == 0:
             return None
-        if not hasattr(self, "DM_r_vmax"):
-            self.DM_r_vmax, self.DM_vmax = get_vmax(
-                self.mass_dm, self.radius[self.dm_mask_sh]
+        if not hasattr(self, "DM_r_vmax_soft"):
+            soft_r = np.maximum(self.softening[self.dm_mask_sh], self.radius[self.dm_mask_sh])
+            self.DM_r_vmax_soft, self.DM_vmax_soft = get_vmax(
+                self.mass_dm, soft_r
             )
-        return self.DM_vmax
+        return self.DM_vmax_soft
 
     @lazy_property
-    def DM_R_vmax(self) -> unyt.unyt_quantity:
+    def DM_R_vmax_soft(self) -> unyt.unyt_quantity:
         """
         Radius for which the maximum circular velocity of dark matter particles
```

Comment:
Perhaps update comment on function to say that it places all particles within softening at a radius equal to the softening.

Reply:
Or say this is Rmax for softened Vmax